lib/tasks/test.js

```diff
@@ -264,5 +271,45 @@ module.exports.browserTest = function(gulp, config) {
 	});
 };
 
+module.exports.pa11yTest = function(gulp, config) {
+	const test = pa11y({
+		log: {
+			debug: console.log.bind(console),
+			error: console.error.bind(console),
+			info: console.info.bind(console)
+		},
+		ignore: ['WCAG2AA.Principle2.Guideline2_4.2_4_1.G1,G123,G124.NoSuchID']
+	});
+
+
+	return new Promise(function(resolve, reject) {
+		config = config || {};
+		let src = (config.pa11yPath !== undefined) ? path.join(process.cwd(), config.pa11yPath) : path.join(process.cwd(), '/demos/local/pa11y.html');
+		log.secondary('Running "pa11y test"...');
+		try{
+			fs.readFile(src, function(err) {
+				if (err === null) {
+					src = 'file://' + src;
+					test.run(src, function(error, result) {
+						console.log('test', 'callback');
+						if (error) {
+							console.log('test', error);
+							reject(error);
+						} else {
+							cliReporter.results(result, src);
+							resolve(result);
+						}
+					});
+				} else {
+					log.primary(`Could not open ${src}`);
+					resolve(err);
+				};
+			});
+		}catch(err){
+			console.log('error', err);
```

Comment:
This should probably be a log.secondaryError. We use the log helper to colour console.logs